tests/daemon/test-machine-id-provider.c

```diff
@@ -31,7 +31,9 @@
 #define HYPHENS_IN_ID 4
 
 #define TESTING_FILE_PATH "/tmp/testing-machine-id"
+#define TESTING_OVERRIDE_FILE_PATH "/tmp/override-testing-machine-id"
 #define TESTING_ID        "04448f74fde24bd7a16f8da17869d5c3\n"
+#define TESTING_OVERRIDE_ID        "04448f74fde24bd7a16f8da17869d5c4\n"
```

Comment:
Might be nice to generate a more-distinct UUID so that assertion failures are clearer, rather than one which differs in only 2 trailing bits!